src/PS1/runtime/EmuCoreLoader.cpp

```
@@ -0,0 +1,79 @@
+QString bundleOrAppDir()
+{
+    return QCoreApplication::applicationDirPath();
+}
+
+QStringList pluginFileNames()
+{
+#if defined(Q_OS_WIN)
+    return {QStringLiteral("qtmesh_ps1core_stub.dll")};
+#elif defined(Q_OS_MACOS)
+    return {QStringLiteral("libqtmesh_ps1core_stub.dylib"),
+            QStringLiteral("qtmesh_ps1core_stub.dylib")};
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_DECLARE_INTERFACE is a macro then please configure it.

(unknownMacro)